source/extensions/filters/network/rocketmq_proxy/well_known_names.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RocketmqProxy {
+
+class RocketmqConstants {
+public:
+  static const std::string CLUSTER_TYPE;
```

Comment:
Please replace it with the idiom used by the Envoy,

```
struct  RocketmqValues {
  const std::string CLUSTER_TYPE{xxxx};
};
using RocketmqConstants = ConstSingleton<RocketmqValues>;
```

Reply:
Nice catch.

Reply:
It has been fixed.